src/Core/Content/DependencyInjection/measurement_system.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <services>
+        <service id="Shopware\Core\Content\MeasurementSystem\Entity\MeasurementDisplayUnitEntity">
+            <tag name="shopware.entity"/>
+        </service>
+
+        <service id="Shopware\Core\Content\MeasurementSystem\Entity\MeasurementSystemEntity">
+            <tag name="shopware.entity"/>
+        </service>
+
+        <service id="Shopware\Core\Content\MeasurementSystem\Entity\ProductMeasurementDisplayUnitEntity">
+            <tag name="shopware.entity"/>
+        </service>
+    </services>
+</container>
```

Comment:
```xml
1. MeasurementSystemEntity

2. MeasurementDisplayUnitEntity

3. ProductMeasurementDisplayUnitEntity
```

Reply:
It would be more logical to have `MeasurementSystemEntity` first in the order. This would better reflect the dependency hierarchy and make the configuration more intuitive to read.